requirements/extra-ml-requirements.txt

```diff
@@ -50,3 +50,5 @@ prophet
 pmdarima
 # Required by mlflow.diviner
 diviner
+# Required by mlflow.tensorflow
```

Comment:
This sound like `mlflow.tensorflow` uses `transformers`, but it doesn't.

Reply:
Only a few tests require transformers. Do we really need to include it in `requirements/extra-ml-requirements.txt`?

Reply:
ok, what about moving transformers into test requirements ?
A test requires it.